includes/class-events-store.php

```diff
@@ -251,7 +258,18 @@ public function remove_multisite_table( $tables_to_drop ) {
 	 * Override cron option requests with data from custom table
 	 */
 	public function get_option() {
+
+		// If this thread has already generated the cron array,
+		// use the copy from local memory. Don't fetch this list
+		// remotely multiple times per request (even from the
+		// object cache).
+		static $cron_array;
+		if ( $cron_array && ! $this->invalid_cache ) {
+			return $cron_array;
+		}
+
 		// Use cached value when available.
+		$this->invalid_cache = false;
```

Comment:
Could rename this to `$is_cache_valid` which would make it's purpose and use a bit clearer.